09-import-css/src/components/todo-list.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import TodoListItem from './todo-list-item'
+import './todo-list.css'
+
+const TodoList = ({todos}) => {
+    const elements = todos.map((item)=>{
```

Comment:
```suggestion
    const elements = todos.map((item) => {
```
Не хватает пробелов вокруг ` => `

Reply:
done